### PR TITLE
LPS-194529 [Impedibug] Fix Missing informations on the Solutions Pages

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Comment by claricebrasil** [comment]
ci:test:ci

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layouts/03_solutions_marketplace/18_openk9/page-definition.json [hunk 1]
```diff
@@ -1498,6 +1498,568 @@
...
+																											"indexed": true
+																										},
+																										"type": "Fragment"
+																									},
+																									{
+																										"definition": {
+																											"fragment": {
+																												"key": "text",
+																												"siteKey": "mkt"
+																											},
```
Comment (juliocezar96) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layouts/03_solutions_marketplace/18_openk9/page-definition.json:1717]:
```suggestion
																												"siteKey": "[$GROUP_KEY$]"
																											},
```

**Review by juliocezar96 — CHANGES_REQUESTED** [review]
Hey, good job! But it has a small adjustment. 🧑‍💻 👀

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @claricebrasil

**Comment by claricebrasil** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by claricebrasil** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:stop

**Comment by claricebrasil** [comment]
resent: #4717

**Pull request closed without merging**